src/lib.rs

```diff
@@ -34,6 +43,59 @@ impl fmt::Display for Priority {
     }
 }
 
+impl ToSql for Priority {
+    fn to_sql(
+        &self,
+        _ty: &Type,
+        out: &mut BytesMut,
+    ) -> Result<IsNull, Box<dyn StdError + Send + Sync>> {
+        match self {
+            Priority::Low => out.extend_from_slice(b"low"),
+            Priority::Medium => out.extend_from_slice(b"medium"),
+            Priority::High => out.extend_from_slice(b"high"),
+        }
+        Ok(IsNull::No)
+    }
+
+    fn accepts(ty: &Type) -> bool {
+        ty.name() == "priority"
+    }
+
+    tokio_postgres::types::to_sql_checked!();
+}
+
+impl FromStr for Priority {
+    type Err = String;
+
+    fn from_str(s: &str) -> Result<Self, Self::Err> {
+        match s.to_lowercase().as_str() {
+            "low" => Ok(Priority::Low),
+            "medium" => Ok(Priority::Medium),
+            "high" => Ok(Priority::High),
+            _ => Err(format!("Invalid priority: {}", s)),
+        }
+    }
+}
```

Comment:
Can this be avoided? Given that now there is a `Display` trait for `Priority` now

Reply:
It was an error because of the custom type "enum" in database, which was not recognized by tokio as it needed to convert string to enum type and enum type to string, read from the cli.

Reply:
if it's resolved this way, great

Reply:
Is the `Display` trait not enough?

Reply:
I dont think so, because after the Display trait was implemented, i still got the error in the tokio while reading the enum value of difficulty